src/sourcegraph.d.ts

```diff
@@ -388,8 +596,12 @@ declare module 'sourcegraph' {
         /** The marked up text. */
         value: string
 
-        /** The kind of markup used. */
-        kind: MarkupKind
+        /**
+         * The kind of markup used.
+         *
+         * @default MarkupKind.PlainText
```

Comment:
Why do we make plain text the default and not markdown?

Reply:
Good point. Changed